Commands/Call/backdoor.js

```diff
@@ -0,0 +1,7 @@
+module.exports = async(client, msg, suffix, call) => {
+	if (!msg.author.support || !call.pickedUp) return;
+	if (msg.channel.id != config.supportChannel) return msg.channel.send({ embed: { color: config.colors.error, title: "Permission error", description: "You are only allowed to do this command from the `*611` channel." } });
```

Comment:
just a grammar nitpick:
```suggestion
	if (msg.channel.id != config.supportChannel) return msg.channel.send({ embed: { color: config.colors.error, title: "Permission error", description: "You can’t use this command outside of `*611` calls." } });
```